tutorials/data_generation_ppi.ipynb

```diff
@@ -239,6 +239,7 @@
     "queries.process(\n",
     "\tprefix = os.path.join(processed_data_path, \"residue\", \"proc\"),\n",
     "\tfeature_modules = [components, contact],\n",
+    "    cpu_count = 8,\n",
```

Comment:
why did you decide to hard code this?

Reply:
Because depending on the number of cpus of the machine of the user, this can get out of control (I run it on Snellius without setting it and it was generating more than 50 files). Setting it to 8 (or whatever small number) ensures that this is the maximum number of files generated. If the user has fewer cpus available, then it generates fewer files but it's fine.